design/Internal/cardsorting.svg

```diff
@@ -1,12 +1,12 @@
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
   <defs>
     <clipPath id="a">
-      <rect width="24" height="24" fill="none"/>
+      
     </clipPath>
   </defs>
   <title>cardsorting</title>
   <g clip-path="url(#a)">
-    <rect width="24" height="24" fill="none" stroke="#000" stroke-width="0" opacity="0.01"/>
+    
   </g>
   <g>
```

Comment:
Can we get rid of these empty groups now? Was that clip path actually useful (is there overflow in any of the other paths or anything like that)?